packages/deployments/contracts/contracts_forge/RelayerFeeRouter.t.sol

```diff
@@ -0,0 +1,205 @@
+    assertEq(address(relayerFeeRouter.connext()), connext2);
+  }
+
+  // Fail if not called by owner
+  function testSetConnextOnlyOwner() public {
+    vm.prank(address(0));
+    vm.expectRevert("Ownable: caller is not the owner");
+    relayerFeeRouter.setConnext(connext2);
+  }
+
+  // ============ send ============
+  // Fail if not called by connext
+  function testSendOnlyConnext() public {
+    vm.prank(address(0));
+    vm.expectRevert(abi.encodeWithSelector(RelayerFeeRouter.RelayerFeeRouter__onlyConnext_notConnext.selector));
+
+    bytes32[] memory transactionIds = new bytes32[](2);
+    transactionIds[0] = "AAA";
+    transactionIds[1] = "BBB";
+
+    relayerFeeRouter.send(remoteDomain, recipient, 1, transactionIds);
+  }
+
+  // Fail if not no amount and no transaction ids
+  function testSendClaimEmpty() public {
+    vm.expectRevert(abi.encodeWithSelector(RelayerFeeRouter.RelayerFeeRouter__send_claimEmpty.selector));
+
+    bytes32[] memory transactionIds = new bytes32[](0);
+
+    relayerFeeRouter.send(remoteDomain, recipient, 0, transactionIds);
+  }
+
+  // Fail if empty recipient
+  function testSendRecipientEmpty() public {
+    vm.expectRevert(abi.encodeWithSelector(RelayerFeeRouter.RelayerFeeRouter__send_recipientEmpty.selector));
+
+    bytes32[] memory transactionIds = new bytes32[](2);
+    transactionIds[0] = "AAA";
+    transactionIds[1] = "BBB";
+
+    relayerFeeRouter.send(remoteDomain, address(0), 1, transactionIds);
+  }
+
+  // Should work
+  function testSend(bytes32[] calldata _transactionIds, uint256 _amount) public {
+    vm.assume(_amount != 0 || _transactionIds.length != 0);
+    bytes memory message = RelayerFeeMessage.formatClaimFees(recipient, _amount, _transactionIds);
+
+    vm.expectCall(home, abi.encodeWithSelector(MockHome.dispatch.selector, remoteDomain, remote, message));
+    vm.expectEmit(true, true, true, true);
+    emit Send(remoteDomain, recipient, _amount, _transactionIds, remote, message);
+
+    relayerFeeRouter.send(remoteDomain, recipient, _amount, _transactionIds);
+  }
+
+  function testSend0Amount(bytes32[] calldata _transactionIds) public {
+    vm.assume(_transactionIds.length != 0);
+    bytes memory message = RelayerFeeMessage.formatClaimFees(recipient, 0, _transactionIds);
+
+    vm.expectCall(home, abi.encodeWithSelector(MockHome.dispatch.selector, remoteDomain, remote, message));
+    vm.expectEmit(true, true, true, true);
+    emit Send(remoteDomain, recipient, 0, _transactionIds, remote, message);
+
+    relayerFeeRouter.send(remoteDomain, recipient, 0, _transactionIds);
+  }
+
+  // ============ handle ============
+  // Should work
+  function testHandle(
+    bytes32[] calldata _transactionIds,
+    uint32 _nonce,
+    uint256 _amount
+  ) public {
+    vm.assume(_amount != 0 || _transactionIds.length != 0);
+    uint64 originAndNonce = (uint64(remoteDomain) << 32) | _nonce;
+    bytes memory message = RelayerFeeMessage.formatClaimFees(recipient, _amount, _transactionIds);
```

Comment:
Because there is no assertion of handle or its effects, it's hard to say if the message is formatting/parsing properly. 

I would add in a separate message test contract with some simple tests that assert you can pull the proper values out of the formatted message